src/MapImportedContent.php

```diff
@@ -170,16 +185,67 @@ protected function castAttribute(string $column, string $attribute, string $mode
             }
 
             if (is_array($callback)) {
-                return (new ValidationPipeline)($value, $callback);
+                return (new CastingPipeline)($value, $callback);
             }
         }
 
         return $value;
     }
 
+    protected function validateAttribute(string $column, string $attribute, string $model, array $row): ?bool
+    {
+        $value = array_key_exists($column, $row) ? $row[$column] : null;
+
+        if (!$this->validators) {
```

Comment:
It looks the casting will not happen if validation array is empty? This is what you want?